src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/visualeditor/VisualEditorPageObject.java

```diff
@@ -83,6 +83,18 @@ public VisualEditorPageObject(WebDriver driver) {
   private WebElement toggler;
   @FindBy(css = ".ve-ce-surface-highlights-focused .ve-ce-focusableNode-highlight")
   private WebElement focusedHighlight;
+  @FindBy(css = ".oo-ui-popupToolGroup-handle")
+  private List<WebElement> toolsList;
+  @FindBy(css = ".oo-ui-tool-title")
+  private List <WebElement> insertMenuTools;
+  @FindBy(css = ".oo-ui-window-frame .oo-ui-labelElement")
+  private List <WebElement> infoboxTemplatesList;
+  @FindBy(css = ".ve-ui-mwParameterPage-field .oo-ui-inputWidget textarea")
+  private List<WebElement> parametersFieldList;
+  @FindBy(css = ".oo-ui-buttonElement-button")
+  private List<WebElement> buttonsList;
+  @FindBy(css = ".oo-ui-popupWidget-body .oo-ui-widget-enabled")
+  private WebElement infoboxPopup;
 
   private By contextMenuBy = By.cssSelector(".ve-ui-contextSelectWidget");
   private By contextEditBy = By.cssSelector(".oo-ui-labelElement");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)